web/text_layer_builder.js

```diff
@@ -321,9 +321,13 @@ var TextLayerBuilder = (function TextLayerBuilderClosure() {
     _bindMouse: function TextLayerBuilder_bindMouse() {
       var div = this.textLayerDiv;
       var self = this;
+      var expandDivsTimer;
       div.addEventListener('mousedown', function (e) {
         if (self.enhanceTextSelection && self.textLayerRenderTask) {
           self.textLayerRenderTask.expandTextDivs(true);
+          if (expandDivsTimer) {
+            window.clearTimeout(expandDivsTimer);
+          }
```

Comment:
Set `expandDivsTimer = null;` after clearing the timeout, otherwise the `if` clause will always be true after activating the timer one time.